specification/authorization/resource-manager/Microsoft.Authorization/2017-10-01-preview/authorization.json

```diff
@@ -718,7 +718,7 @@
           "readOnly": true,
           "description": "The role definition name."
         },
-        "type": {
+        "roletype": {
           "type": "string",
```

Comment:
From what we talked offline, this should stay as `type`

Reply:
since there are two fields with name "type" in the response that is sent by the service.
we cant name this as type,because we would leave out the property with value(for example) type which corresponds to "CUSTOM Role"